sourcemash/frontend/frontend.py

```diff
@@ -14,6 +14,6 @@ def survey():
     return redirect('http://goo.gl/forms/wmnCAcbrLp')
 
 
-@bp.app_errorhandler(404)
-def page_not_found(e):
-    return redirect('/#404')
+@bp.route('/<path:dummy>')
+def fallback(dummy):
+    return redirect('/#' + dummy)
```

Comment:
Any url will retry with the # in front. If our app_router still doesn't find a route, then it will go to our JS 404 template. This is nicer than handling 404s in Flask because we can now link someone to sourcemash.com/categorizer!